### DISCUSSION TITLE
Set Time/date by serial command

### DISCUSSION THREAD

**Discussion by Cloolalang** [arendst/Tasmota#19981] [Q&A] [answered]
Hi all, I have setup a Tasmota device connected via uart radio link (Reyax Lora radios).

I was wondering if there is any way to set the device RTC to a time/date with a serial command?
 (My device is has no WiFi /IP connectivity as its in a remote location).

**Comment by sfromis** [comment, marked as answer]
There is the command `Time` to set an epoch value (in seconds), and all commands can be executed via serial.

**Reply by Cloolalang** [reply]
Thanks sfromis! I will give it a try! I really appreciate it!
